### ISSUE TITLE
No hem allowance on bruce V2.0.1 and 2.0.2

### ISSUE THREAD

**Issue by grimlokason** [freesewing/freesewing#106]
Hello,

I tried  to make a pattern of bruce, but it seem that the hem allowance is not anymore here on 
side and inset, while it's still here on back.  

My recipe :

```json
{
  "handle": "lpoct",
  "name": "1st try latex",
  "notes": "",
  "recipe": {
    "settings": {
      "sa": 4.9,
      "complete": true,
      "paperless": false,
      "locale": "en",
      "options": {
        "stretch": 0.09,
        "legBonus": 0.09
      },
      "units": "metric",
      "measurements": {
        "hipsCircumference": 800,
        "upperLegCircumference": 490,
        "hipsToUpperLeg": 190
      }
    },
    "pattern": "bruce",
    "model": "idcke"
  },
  "created": "2019-09-06T23:17:57.529Z"
}
```

the result : 
[pattern-a4(4).pdf](https://github.com/freesewing/freesewing/files/3613196/pattern-a4.4.pdf)

It also seem that the side part is upside down,  ( the fitting was better with the largest part on the leg and the short on the hip.) 

Thank for your pattern !

**Comment by joostdecock** [comment]
Well spotted @grimlokason thanks for reporting this :+1: 

You are correct, the hem allowance is missing. We'll get to fixing that ASAP.

However, the side part is not upside down. If you find the legs too narrow, you might want to tweak the **Leg stretch** setting in Bruce.

**Comment by joostdecock** [comment]
This has been fixed in v2.0.3 which is now deployed on freesewing.org
